index.html

```diff
@@ -42,7 +43,7 @@
             </div>
             <div class="nav_desktop">
                 <span class="crossBtn">&times;</span>
-                <a onclick="hideMenu()" class="nav_link func_01" href="../index.html">Home</a>
+                <a onclick="hideMenu()" class="nav_link nav_home" href="../index.html"><img src="./images/logo.png" alt="logo" class="logo_img"><h3>Pakistan Watch Week</h3></a>
```

Comment:
- [ ] You did a great job, @HAMMAS-SALEEM. Nice one:+1:
Above this element, the top-most element, can you make them a link, please? I am of the opinion that it will make more sense to add the `cursor: pointer;` CSS declaration to the items. They are links of course. What do you think?
![image](https://user-images.githubusercontent.com/45668340/147160682-ed24af7b-5e5c-4f66-95c5-3df0d97d9d92.png)